README.md

```diff
@@ -65,20 +65,31 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 > **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler.
 
 ## ✅ Getting started
-
 ### 🧰 Installing `ilab`
 
-1. If you are on Fedora Linux, install C++, Python 3.9+, and other necessary tools by running the following command:
+1. You first need to install Python version 3.9+ and C++.
 
-   ```shell
-   sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
-   ```
+   #### Installing packages on Linux (Fedora)
+
+   * Install C++, Python 3.9+, and other necessary tools by running the following command:
+
+      ```shell
+      sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
+      ```
+
+      Optional: If `g++` is not found, try `gcc-c++` by running the following command:
+
+      ```shell
+      sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
+      ```
+
+   #### Installing packages on MacOS*
 
-   Optional: If `g++` is not found, try `gcc-c++` by running the following command:
+   * Install C++, Python 3.9+, and other necessary tools by running the following command:
 
-     ```shell
-     sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
-     ```
+      ```shell
+      brew install g++ gcc make pip python3 python3-devel python3-GitPython
```

Comment:
```suggestion
      xcode-select --install
```

Xcode command line tools includes C++ compiler, Python 3.9.

Reply:
The CLI folks were trying to avoid x-code becuase it installs a large amount of dependences and they thought it was easier to download the dependencies on their own.

Reply:
A macOS user would need to install brew. It is not standard. And many brew formulas require Xcode command line tools to be installed. I think the standard brew installation process asks you to install Xcode command line tools. 

Also:

```shell
➜  ~ brew info pip python3-devel python3-GitPython
Error: No available formula with the name "pip". Did you mean pipx, pig, pit, pop, pup, php, pcp, zip, sip or vip?
pip is part of the python formula:
  brew install python

Error: No available formula with the name "python3-devel". Did you mean python-dateutil?

Error: No available formula with the name "python3-gitpython". Did you mean python-ply?                                                     
```

Reply:
https://docs.brew.sh/Installation#macos-requirements indicates installing Xcode command lines tools is a requirement.